### PR TITLE
Use gs as alias for grass.script in doc

### PR DESCRIPTION
**Pull request by wenzeslaus**
Both grass and gscript aliases (import grass.script as ...) were replaced by gs
which is shorter but still refering to grass.script.

Changes documentation (.html and .rst), examples (files, wxGUI/pyedit),
grass.script test, and how grass.script is accessible in GUI Python shell (wxGUI/pyshell).
Graphical modeler code generation does not use alias and imports specific objects.
Does not change any of the 'live' source code which still uses mixture of

    import grass.script as grass

and

    import grass.script as gscript

Also touches on couple places to clarify references to grass.script
and to GRASS GIS in general.

### CODE REVIEW

**Comment by neteler** [comment]
@wenzeslaus I would recommend a backport, otherwise future backporting will become painful.
Any objections?

File: lib/python/script/setup.py [hunk 1]
```diff
@@ -59,22 +59,22 @@
...
-    import grass.script as gscript
+    import grass.script as gs
     import grass.script.setup as gsetup
 
     # launch session
     rcfile = gsetup.init(gisbase, gisdb, location, mapset)
 
     # example calls
-    gscript.message('Current GRASS GIS 7 environment:')
-    print gscript.gisenv()
```
Comment (petrasovaa) [inline lib/python/script/setup.py]: Change to py3 style print

File: lib/python/script/testsuite/test_script_raster.py [hunk 2]
```diff
@@ -29,15 +29,15 @@ def tearDownClass(cls):
                       flags='f')
 
     def test_raster_what(self):
-        res = gscript.raster_what(self.raster, [self.coords])[0]
+        res = gs.raster_what(self.raster, [self.coords])[0]
         self.assertEquals(int(res[self.raster]['value']), 100)
 
-        res = gscript.raster_what(self.raster, [self.coords],
+        res = gs.raster_what(self.raster, [self.coords],
```
Comment (petrasovaa) [inline lib/python/script/testsuite/test_script_raster.py]: indentation

**Comment by wenzeslaus** [comment]
@neteler In this case which is mainly documentation and examples, I don't have a strong opinion either way, but this should be eventually followed by another change to change this in `scripts/`. That change could break things, so I would be against the backport in that case as it is not a fix.

File: gui/wxpython/gui_core/pyedit.py [hunk 3]
```diff
@@ -121,14 +121,14 @@ def module_template():
 def cleanup():
 """)
     output.write(
-        r"""    gscript.run_command('g.remove', flags='f', type='raster',
+        r"""    gs.run_command('g.remove', flags='f', type='raster',
```
Comment (petrasovaa) [inline gui/wxpython/gui_core/pyedit.py]: indentation

File: gui/wxpython/gui_core/pyedit.py [hunk 4]
```diff
@@ -146,16 +146,16 @@ def script_example():
...
 def main():
     input_raster = 'elevation'
     output_raster = 'high_areas'
-    stats = gscript.parse_command('r.univar', map='elevation', flags='g')
+    stats = gs.parse_command('r.univar', map='elevation', flags='g')
     raster_mean = float(stats['mean'])
     raster_stddev = float(stats['stddev'])
     raster_high = raster_mean + raster_stddev
-    gscript.mapcalc('{r} = {a} > {m}'.format(r=output_raster, a=input_raster,
+    gs.mapcalc('{r} = {a} > {m}'.format(r=output_raster, a=input_raster,
```
Comment (petrasovaa) [inline gui/wxpython/gui_core/pyedit.py]: indentation

**Pull request merged**